backend/coins/btc/handlers/handlers.go

```diff
@@ -348,18 +348,31 @@ func (handlers *Handlers) postAccountTxProposal(r *http.Request) (interface{}, e
 	}, nil
 }
 
+type jsonFeeTarget struct {
```

Comment:
You don't have to define it globally. More recent Go versions allow for a type to be defined within a function.

Reply:
Wow this changes everything! When did that happen? Pretty sure that was not possible a year ago.

Reply:
Done